azure-pipelines.yml

```diff
@@ -1,9 +1,49 @@
 # Adapted from https://github.com/pandas-dev/pandas/blob/master/azure-pipelines.yml
 jobs:
+- job: linting
+  displayName: Linting
+  pool:
+    vmImage: ubuntu-16.04
+  steps:
+    - bash: echo "##vso[task.prependpath]$CONDA/bin"
+      displayName: Add conda to PATH
+    - bash: sudo chown -R $USER $CONDA
+      displayName: Take ownership of conda installation
+    - bash: conda create --name flake8_env --yes flake8
+      displayName: Install flake8
+    - bash: |
+        source activate flake8_env
+        ./build_tools/circle/flake8_diff.sh
```

Comment:
You have two lint commands here... Should these be together in a single script to make it easy to change both circle and Azure runs?

Reply:
I would be okay with combining them. What do you think of naming the combined script "linting.sh"?

Reply:
> I would be okay with combining them. What do you think of naming the combined script "linting.sh"?

sounds good to me.